homeassistant/components/braviatv/config_flow.py

```diff
@@ -157,7 +157,7 @@ async def async_step_init(
         content_mapping = await self.hass.async_add_executor_job(
             braviarc.load_source_list
         )
-        self.source_list = {item: item for item in [*content_mapping]}
+        self.source_list = {item: item for item in (*content_mapping,)}
```

Comment:
So why are we making a copy to then iterate over it to begin with here ? 

I also don't think that the new syntax is any more readable, if at all. If we want to make a copy we can do `tuple(content_mapping)` or `list(content_mapping)` but neither makes sense because we iterate over it right away and throw it out. 

This looks like it should work:

```suggestion
        self.source_list = {item: item for item in content_mapping}
```

Reply:
The pylint suggestion is only to use a tuple instead of list, for consistency.
But indeed, it does make more sense to use `content_mapping` directly here.